en/docs/administer/managing-users-and-roles/managing-user-roles.md

```diff
@@ -108,6 +108,11 @@ Follow the instructions below to create the `creator`, `publisher`, and `subscri
 
     These roles do not have any permissions assigned to it, but it is used to manage the visibility of the corresponding service provider that is created in the format of `'<username>_<applicationName>_PRODUCTION'` within the Key Manager. The created service provider is only visible to users with the latter mentioned role that has been generated automatically. Only if a user with admin privileges assigns the latter mentioned role to a user, will that user be able to view the details of the service provider that is created per application.
 
+!!! warning
+    As a limitation, when a user is deleted and create another user with same username, previous users applications will be visible in the Dev portal applications listing page. 
+    This is due to a limitation in the database design since use username as a key to store the Applications. 
```

Comment:
lets remove this line

Reply:
Fix https://github.com/wso2/docs-apim/pull/2550/commits/3af9262137499afc9ab91481a7acb23857cddc80